server/src/entity/User.ts

```diff
@@ -9,6 +9,8 @@ import { IsEmail, IsUrl, MaxLength, MinLength } from "class-validator";
 import { Field, ObjectType, InputType } from "type-graphql";
 import City from "./City";
 
+export type Role = "admin" | "user" | "contributor";
```

Comment:
can you delete this code, which is not relevant on this branch (but will be on the feature-role-management branch)?